docs/user-guide/longitudinal-profile.md

```diff
@@ -0,0 +1,87 @@
+# Longitudinal Profile
+
+*******************
+
+The **Longitudinal Profile** tool allows the user to analyze the altitude or depth data present in digital terrain models or bathymetric surveys and displays the found values in an interactive graphic profile.
+
+!!! note
+    By default, the **Longitudinal Profile** plugin is ready to be configured for [application contexts](application-context.md#configure-plugins), and is not available in the default plugin configuration.
+    The plugin works only if the [WPS process](gs-wps-longitudinal-profile-2.24-SNAPSHOT.zip) is configured on the [GeoServer](http://geoserver.org/).
+
+By clicking the **Longitudinal Profile** <img src="../img/button/long-profile-button.jpg" class="ms-docbutton"/> button, available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar), a drop down menu opens and the user can select the way to calculate altitude and distance traveled values, choosing between three options:
+
+<img src="../img/longitudinal-profile/dropdown-menu-long-profile.jpg" class="ms-docimage"/>
+
+* The **Draw line** to draw a line directly on the map through <img src="../img/button/drawing-line.jpg" class="ms-docbutton"/> button
+
+<img src="../img/longitudinal-profile/drawing-line.gif" class="ms-docimage"/>
+
+* The **Load file** to import linear profile as `JSON`, `ShapeFile` or `DXF`, selected from file system, through <img src="../img/button/import-file.jpg" class="ms-docbutton"/> button
```

Comment:
```suggestion
* In alternative it is also possible to import a linear profile as a vector file (available formats for this are `GeoJSON`, `ShapeFile` or `DXF`),  through <img src="../img/button/import-file.jpg" class="ms-docbutton"/> button
```

Reply:
Alternatively it is also possible to import a linear profile as a vector file (available formats for this are `GeoJSON`, `ShapeFile` or `DXF`),  through <img src="../img/button/import-file.jpg" class="ms-docbutton"/> button